src/llmcompressor/transformers/sparsification/compressed_tensors_utils.py

```diff
@@ -124,6 +125,7 @@ def save_pretrained_wrapper(
             quantization_format: Optional[str] = None,
             save_compressed: bool = True,
             skip_compression_stats: bool = False,
+            no_sparse_compression: bool = False,
```

Comment:
Can we add a test case to cover this case?

Reply:
Added